examples/rust-gcoap/Makefile

```diff
@@ -34,10 +34,9 @@ DEVELHELP ?= 1
 # Change this to 0 show compiler invocation lines by default:
 QUIET ?= 1
 
-# Add 8k extra stack: The Rust examples take more of it than gcoap expects,
-# presumably because the example use the standard library's sting formatting
-# instead of one of the more optimized formatters.
-CFLAGS += -DGCOAP_STACK_SIZE='(THREAD_STACKSIZE_DEFAULT+DEBUG_EXTRA_STACKSIZE+sizeof(coap_pkt_t)+8192)'
+# Add 16k extra stack: The Rust examples take more of it than gcoap expects,
```

Comment:
could you try it with e.g. 12k, to see if it still fits on the sam* devices? I feel those might be common enough to justify breaking the beauty of 2^x :)

Reply:
Indeed it fits, and shaved 1k of the main thread too. generate-Makefile.ci in progress, should suffice for them!

Reply:
```suggestion
# Add 12k extra stack: The Rust examples take more of it than gcoap expects,
```